docs/testing/debug-office-add-ins-on-ipad-and-mac.md

```diff
@@ -1,19 +1,36 @@
 ---
 title: Debug Office Add-ins on iPad and Mac
 description: ''
-ms.date: 01/25/2018
+ms.date: 03/21/2018
 ---
 
 # Debug Office Add-ins on iPad and Mac
 
 You can use Visual Studio to develop and debug add-ins on Windows, but you can't use it to debug add-ins on the iPad or Mac. Because add-ins are developed using HTML and Javascript, they are designed to work across platforms, but there might be subtle differences in how different browsers render the HTML. This article describes how to debug add-ins running on an iPad or Mac. 
 
-## Debugging with Vorlon.JS 
+## Debugging with Safari Web Inspector on a Mac
 
-Vorlon.JS is a debugger for web pages, similar to the F12 tools. It is designed to work remotely and it enables you to debug web pages across different devices. For more information, see the [Vorlon website](http://www.vorlonjs.com).  
+You can debug an Office add-in using Safari Web Inspector. 
 
+To enable debugging Office add-ins on Mac, you must have Mac OS High Sierra AND Mac Office Version: 16.9.1 (Build 18012504) or later. If you don't have an Office Mac build, you can get one by joining the [Office 365 Developer program](https://aka.ms/o365devprogram).
```

Comment:
suggest replacing the "To enable debugging Office add-ins..." at the start of the sentence with "To be able to debug Office Add-ins...."

i.e., change wording as suggested and capitalize "Add-ins"

Reply:
fixed